assets/scss/components/_buttons.scss

```diff
@@ -16,10 +17,10 @@
   margin-left: 1.25rem;
 }
 
-@include media-breakpoint-up(lg) {
-  #mode {
-    margin-left: 0.5rem;
-    margin-right: 0.25rem;
+@include media-breakpoint-up(md) {
+  .toggle-theme-mode {
+    margin-left: 1.125rem;
+    margin-right: -0.375rem;
```

Comment:
These are very specific values, did we test them on multiple screen sizes?

Reply:
I tested it on a mobile device when the languages menu is shown inside the sidebar. Then I add margin to the changing theme button 
![image](https://user-images.githubusercontent.com/29014573/201864849-da242ef7-d7c6-4aa1-8d7c-eb56c89e2dd6.png)

However I don't need this and instead added margins to the languages button